components/elm/src/main/elm_varctl.F90

```diff
@@ -508,6 +508,7 @@ module elm_varctl
    ! land river two way coupling
    !----------------------------------------------------------
    logical, public :: use_lnd_rof_two_way = .false.
+   integer, public :: lnd_rof_coupling_nstep = 6
```

Comment:
Let's not initialize it to `0` and have a check to ensure that a valid value is read from the `lnd_in`.

Reply:
Do you mean initialize `lnd_rof_coupling_nstep` as 0?

Reply:
Add the check for `lnd_rof_coupling_nstep` at https://github.com/E3SM-Project/E3SM/blob/2a1ef482627fbb2e697158908fbb5e58e8d8620d/components/elm/src/main/controlMod.F90#L510

Reply:
Done.